mlflow/models/evaluation/base.py

```diff
@@ -496,33 +634,17 @@ def evaluate(
                              a nested dictionary whose key is the evaluator name.
     :return: An :py:class:`mlflow.models.evaluation.EvaluationDataset` instance containing
              evaluation results.
+
+    The default evaluator supports the 'regressor' and 'classifer' `model_type`s. The available
+    `evaluator_config` options for the default evaluator include:
+     - log_model_explainability: The number of rows to use for calculating model explainability.
+     - explainality_algorithm: A string to specify the shap explainer algorithm. If not set, it will
```

Comment:
Do we want to explain what the mapping is or the location in the code that this default mapping is defined at?

Reply:
The default algorithm behavir depends on shap Explainer implementation. I refine the doc like:
```
If not set, it will run `shap.Explainer` with "auto" algorithm, which choose the best fit explainer according to the model.
```

Reply:
Ah yes the internal deciding logic within SHAP. sounds good